src/SSHDebugPS/UI/ContainerPickerDialogWindow.xaml

```diff
@@ -176,24 +278,25 @@
     <!-- Main layout grid -->
     <Grid Margin="12">
         <Grid.RowDefinitions>
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="Auto" />
-            <RowDefinition Height="*" />
-            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" /> <!--0:Connectionlabel-->
+            <RowDefinition Height="Auto" /> <!--1:Connection combo box/add new connection-->
+            <RowDefinition Height="Auto" /> <!--4:Advanced Settings-->
```

Comment:
><!--4:Advanced Settings--> [](start = 44, length = 26)

These comments look like helpful reminders for what content is in which row, but the row numbers and content descriptions don't seem to match up (for example. shouldn't the ContainerList be row 4?).  Did I misinterpret?